ColorPairUnitTest.py

```diff
@@ -0,0 +1,13 @@
+from ColorPairGenerator import get_color_from_pair_number
+from ColorPairGenerator import get_pair_number_from_color
+
+def test_number_to_pair(pair_number,

```

Comment:
Try including the expectation in the test name. See if new tests emerge.

Reply:
The names has been changed to expectation but not pushed to directory

def **test_pairNumber_to_colorPair**(pair_number,
                        expected_major_color, expected_minor_color):
  major_color, minor_color = get_color_from_pair_number(pair_number)
  assert(major_color == expected_major_color)
  assert(minor_color == expected_minor_color)


def **test_colorPair_to_pairNumber**(major_color, minor_color, expected_pair_number):
  pair_number = get_pair_number_from_color(major_color, minor_color)
  assert(pair_number == expected_pair_number)